lib/utils/update.js

```diff
@@ -0,0 +1,43 @@
+const pkg = require('../../package.json')
+const updateNotifier = require('update-notifier')
+const boxen = require('boxen')
+
+module.exports = () => {
+  const notifier = updateNotifier({
+    pkg,
+    updateCheckInterval: 1000 * 60 * 60
+  })
+
+  if (!notifier.update) {
+    return
+  }
+
+  // Depending on running OS show appropriate instructions:
+  let instructions = 'If you\'ve installed data tool using our executable binary then follow instructions below:\n'
```

Comment:
I don't like hardcoding the instructions like
```javascript
instructions = "..."
if (...) instructions += "..."
if (...) instructions += "..."
```

The code looks not good and It is hard to maintain instructions in the future.

Could you extract instructions in the 
```javascript
instructions = {
for_linux : "...",
for_win : "...",
for_mac: "..."
```
structure (could be in the separate file) and use it in the conditions like `if (...) console.log(instructions.for_win)`

Reply:
@anuveyatsu

Reply:
@AcckiyGerman ok, thanks, think it is a good suggestion!